src/codegen/sys/statics.rs

```diff
@@ -6,7 +6,7 @@ pub fn begin(w: &mut Write) -> Result<()> {
     let v = vec![
         "",
         "#![allow(non_camel_case_types, non_upper_case_globals, non_snake_case)]",
-        "#![cfg_attr(feature = \"cargo-clippy\", allow(approx_constant, type_complexity, unreadable_literal))]",
+        "#![cfg_attr(feature = \"cargo-clippy\", allow(clippy::approx_constant, clippy::type_complexity, clippy::unreadable_literal))]",
```

Comment:
We have to switch to 1.31 anyway already, @GuillaumeGomez has that on his list already. Major parts of the ecosystem already don't compile with pre-1.31 anymore, including things like `cc`.

For GStreamer I already switched to 1.31 because of that.

Reply:
Thanks for info, then I remove feature here too.